### DISCUSSION TITLE
Certain keywords show font weight of bold.

### DISCUSSION THREAD

**Discussion by apoorv569** [emacs-tree-sitter/elisp-tree-sitter#158] [Q&A]
I am using `Doom Emacs`, I installed this plugin and configured it like this,
```
;; Tree sitter
(use-package! tree-sitter
  :config
  (require 'tree-sitter-langs)
  (global-tree-sitter-mode)
  (add-hook 'tree-sitter-after-on-hook #'tree-sitter-hl-mode))
```

It does work, but for some reason certain keywords like a call to a class method show up as font weight of bold, when they should show as normal weight.
![2021-05-26_20-25](https://user-images.githubusercontent.com/53279454/119684997-82627700-be34-11eb-9709-820466dbf57c.png)
As you can see all `#include` statements show as bold here and,

![2021-05-26_20-26](https://user-images.githubusercontent.com/53279454/119685069-927a5680-be34-11eb-82d9-4441da7ea200.png)
These methods also show as bold.

Is there a way to make them normal weight, without turning off the bold weight globally?

**Comment by rirze** [comment]
Take a look at the documentation (https://ubolonton.github.io/emacs-tree-sitter/syntax-highlighting/customization/)
